docs/sequences/index.rst

```diff
@@ -21,6 +35,62 @@ Sequences and Looping
         }
     }
 
-6) etc.
+6) generators, iterators
+
+In python, a simple generator is written like this:
+
+.. code-block:: python
+
+    # python
+    def gen():
+        for i in range(5):
+            yield i
+
+And you can iterate over it like a normal list but w/o a known index.
+
+This can be replicated in kotlin like this:
+
+.. code-block:: kotlin
+
+    val gen = sequence<Int> {
+        for (i in 0..4) {
+            yield(i)
+        }
+    }
+
+Beware: there is a huge difference between python's generators and kotlin's.
+Python's generators are treated as an instance, whilst the kotlin ones are w/o a state.
+What does that mean?
+Iterating in a kotlin generator, then stopping the iteration and starting a new one doesn't continue where it left off.
+
+To accomplice the behaviour we want (keeping a state) we have to save the iterator of that generator, and not the generator itself.
```

Comment:
"accomplish"

Reply:
Lmao, as you can understand I'm not a native English speaker
So some small mistakes slipped by